src/app/pages/message/message.component.html

```diff
@@ -1,22 +1,24 @@
-<div class="messages-page">
-    <p class="messages"> All Messages </p>
-    <div class="table">
-        <div class="date" Date>
-            <h3>Date</h3>
-            12/24
-            12/21
-            12/80
-        </div>
-        <div class="sender" Sender>
-            <h3>Sender</h3>
-            john
-            doe
-            yeah
-        </div>
-        <div class="subject" Subject>
-            <h3>Subject</h3>
-            blah blah blah
-            blah
-            blah blah
-        </div>
-    </div>
\ No newline at end of file
+<table class="table">
+    <tbody>
+        <tr>
+            <td class="column"> Date&nbsp;</td>
```

Comment:
If you're using `<tbody>` it's usually because you're also using `<thead>` which will automatically treat that row as a header section and bold the text and what not so you don't need this kind of column header row.